troposphere/static/js/components/modals/image_version/scripts/ScriptMultiSelectAndCreate.react.js

```diff
@@ -154,7 +154,10 @@ define(function (require) {
           </div>
           {this.renderChosenSearchSelect()}
           <div className="form-group clearfix">
-            <button onClick={this.onEditChange} type="button" className="btn btn-default btn-sm pull-right">{showFormButtonText}</button>
+            <button onClick={this.onEditChange} type="button"
+                    className="btn btn-default btn-sm pull-right">
+                    {showFormButtonText}
+            </button>
```

Comment:
If change was made in ScriptMultiSelectAndCreate its probably also necessary in LicenseMultiSelectAndCreate?

see https://github.com/iPlantCollaborativeOpenSource/troposphere/blob/image_creator_style_fix/troposphere/static/js/components/modals/image_version/licenses/LicenseMultiSelectAndCreate.react.js#L158-L160